src/api/sources.js

```diff
@@ -0,0 +1,20 @@
+import { SERVER } from 'constants.js';
+
+
+const clientList = {
+  get: (userID) => fetch(`${SERVER}/clients?userID=${userID}`),
+  post: (data) => fetch(
+    `${SERVER}/clients`,
+    {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(data),
+    },
+  ).then(() => data),
```

Comment:
I don't think post processing should go here, although I see what you're doing, it's kinda cool to make it look like JSON server actually returned something. 😂